resources/views/platform/components/game/current-progress/progress-bar.blade.php

```diff
@@ -0,0 +1,57 @@
+@props([
+    'totalAchievementsCount' => 0,
+    'numEarnedSoftcoreAchievements' => 0,
+    'numEarnedHardcoreAchievements' => 0,
+])
+
+<?php
+$hardcoreProgressWidth = 0;
+$softcoreProgressWidth = 0;
+
+// Never divide by zero.
+if ($totalAchievementsCount > 0) {
+    $hardcoreProgressWidth = ($numEarnedHardcoreAchievements / $totalAchievementsCount) * 100;
+    $softcoreProgressWidth = ($numEarnedSoftcoreAchievements / $totalAchievementsCount) * 100;
+}
+
+$completionPercentage = sprintf("%01.0f", floor($hardcoreProgressWidth + $softcoreProgressWidth));
```

Comment:
Probably intentional, but the hardcore percentage is not available in mixed mode. Hovering over the hardcore part of the bar still shows the overall completion percentage.

![image](https://github.com/RetroAchievements/RAWeb/assets/32680403/9697eb05-7b54-4fe3-8baa-78c94e600f33)

Reply:
Agree with your thoughts that there's room for improvement here. In latest, two percentages are now shown in mixed mode if the numbers are sufficiently (>= 9%) far apart:

![Screenshot 2023-08-26 at 12 59 47 PM](https://github.com/RetroAchievements/RAWeb/assets/3984985/b730f4af-f7b3-4407-9412-984a78152a0a)

Reply:
Should the percentages for pure hardcore/pure softcore be color coded too?

Reply:
Good call, done in https://github.com/RetroAchievements/RAWeb/pull/1785/commits/72d211ce105c66751c362b5fdee25e5ee4c06599.

Reply:
Minor: There's no hardcore percentage for a Completed game with some hardcore progress.

Reply:
Are the two percentages within 9% of each other? There may not be enough space to render the text. I'm having difficulty reproducing the issue:

![Screenshot 2023-08-28 at 5 46 13 PM](https://github.com/RetroAchievements/RAWeb/assets/3984985/fcd6e487-79a9-442b-93fd-4b89a1a6e1dc)

Reply:
![image](https://github.com/RetroAchievements/RAWeb/assets/32680403/e4e9918a-a910-445d-a310-54d0cd882b9a)

16/18=88%

Reply:
Found the bug. Addressed in https://github.com/RetroAchievements/RAWeb/pull/1785/commits/fec74c83ac1511f3c7c690f2982fb24064ff230f.

The "100%" will not show because it is redundant, but your hardcore progress percentage should now appear.